### PR TITLE
GH-3306 added transactions routes

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Handles #3306 

Changes proposed in this pull request:
- Routing for remaining transaction routes

### CODE REVIEW

File: Multisig/UI/App/NavigationRouter.swift [hunk 1]
```diff
@@ -173,7 +173,7 @@ class ExtendedNavigationRouter: NavigationRouter {
                 chainId: safeAddress.chainId
             )
             return route
-        case "/transactions/queue":
+        case "/transactions/queue", "/transactions", "/transactions/messages":
```
Comment (elgatovital) [inline Multisig/UI/App/NavigationRouter.swift:178]: we don't have messages in the app, why supporting this route and forwarding it to tx list?

Comment (DmitryBespalov) [reply Multisig/UI/App/NavigationRouter.swift:178]: here forwarding is to the closest possible screen, thus handling that specific URL as well. This way we handle all of the URLs that are present in the safe web app

Comment (MouazAlzahabi) [reply Multisig/UI/App/NavigationRouter.swift:178]: @DmitryBespalov Could you please just add a TODO to support route to messages if we did it in future ?

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/App/NavigationRouter.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/UI/App/NavigationRouter.swift:176]: I think there is no need to forward something that is not available in the app

Comment (DmitryBespalov) [reply Multisig/UI/App/NavigationRouter.swift:178]: wouldn't this be found anyway?

Comment (DmitryBespalov) [reply Multisig/UI/App/NavigationRouter.swift:178]: can add the explanatory comment, but todo is too much now, as implementation of this is not planned yet

**Pull request merged**